dask/async.py

```diff
@@ -488,12 +488,10 @@ def fire_task():
                 task = dsk[key]
                 _execute_task(task, data)  # Re-execute locally
             else:
-                raise type(res)('\nRemote Exception:\n'
-                              + '-----------------\n'
-                              + str(res) + '\n\n'
-                              + 'Traceback:\n'
-                              + '----------\n'
-                              + tb)
+                print("Traceback:\n"
+                      "----------")
+                print(tb)
```

Comment:
That would also work, but still seems mildly complicated (`Remote*` version for every exception). This seems simple enough to me:

```python
try:
    my_array.compute()
except RemoteError as e:
    orig_err = e.err
    ...
```
Could even re-raise after catching to use the builtin try/except dispatching.